Python/initconfig.c

```diff
@@ -1986,6 +1989,28 @@ config_init_perf_profiling(PyConfig *config)
 
 }
 
+static PyStatus
+config_init_remote_debug(PyConfig *config)
+{
+#ifndef Py_REMOTE_DEBUG
+    config->remote_debug = 0;
+#else
+    int active = 1;
+    const char *env = Py_GETENV("PYTHON_DISABLE_REMOTE_DEBUG");
+    if (env) {
+        active = 0;
+    }
+    const wchar_t *xoption = config_get_xoption(config, L"disable-remote-debug");
```

Comment:
As above, all the other X options use underscores so this should probably be 
```suggestion
    const wchar_t *xoption = config_get_xoption(config, L"disable_remote_debug");
```